README.md

```diff
@@ -8,6 +8,8 @@
 
 Orbit is an MVI framework for Kotlin and Android which we use at [Babylon Health](https://www.babylonhealth.com).
 
+[![slack logo](images/slack-logo-icon.png)](https://kotlinlang.slack.com/messages/CPM6UMD2P)
```

Comment:
the link like this isn't super helpful... what if I don't have a kotlinlang slack account - it doesn't help me create one which would be useful